### PR TITLE
docs: disable properly galera-autofix during an upgrade

### PR DESCRIPTION
**Pull request by nqb**
# Description
Disable properly galera-autofix during an upgrade

# Impacts
Cluster upgrade

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
In a cluster, when I disable a service from GUI (like galera-autofix) using VIP of the cluster, I see `pf.conf` is updated on all nodes but: server which own the VIP has systemd service disabled, other servers still have service enabled in systemd. Is it an expected behavior ?

I tested on Debian.

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 1]
```diff
@@ -1125,6 +1124,20 @@ Once this is done, restart `pfmon` or `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
+===== Disabling the automatic clustering resolution service (for PF versions 10.0 and later)
+
+You should disable `galera-autofix` service in configuration to prevent resolution of cluster issues during upgrade.
```
Comment (julsemaan) [inline docs/PacketFence_Upgrade_Guide.asciidoc]:
```suggestion
You should disable the `galera-autofix` service in the configuration to disabled the automated resolution of cluster issues during the upgrade.
```

File: docs/cluster/appendix.asciidoc [hunk 2]
```diff
@@ -94,7 +94,7 @@ First, ensure you have taken backups of your data. We highly encourage you to pe
 
 The PacketFence clustering stack has a mechanism that allows configuration conflicts to be handled accross the servers. This will come in conflict with your upgrade, so you should disable it.
 
-In order to do, so go in _Configuration->System Configuration->Maintenance_ and disable the _Cluster Check_ task.
+In order to do, go in _Configuration->System Configuration->Maintenance_ and disable the _Cluster Check_ task.
```
Comment (julsemaan) [inline docs/cluster/appendix.asciidoc]:
```suggestion
In order to do so, go in _Configuration->System Configuration->Maintenance_ and disable the _Cluster Check_ task.
```

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 3]
```diff
@@ -1125,6 +1124,20 @@ Once this is done, restart `pfmon` or `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
+===== Disabling the automatic clustering resolution service (for PF versions 10.0 and later)
```
Comment (julsemaan) [inline docs/PacketFence_Upgrade_Guide.asciidoc]:
```suggestion
===== Disabling galera-autofix (for PF versions 10.0 and later)
```

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 4]
```diff
@@ -1544,15 +1540,18 @@ Once this is done, restart `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
-====== Restart the galera-autofix service
+===== Reactivate the automatic clustering resolution service
```
Comment (julsemaan) [inline docs/PacketFence_Upgrade_Guide.asciidoc]:
```suggestion
===== Reactivate galera-autofix
```

File: docs/cluster/appendix.asciidoc [hunk 5]
```diff
@@ -110,6 +110,20 @@ Once this is done, restart `pfmon` or `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
+==== Disabling the automatic clustering resolution service (for PF versions 10.0 and later)
```
Comment (julsemaan) [inline docs/cluster/appendix.asciidoc]:
```suggestion
==== Disabling galera-autofix (for PF versions 10.0 and later)
```

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 6]
```diff
@@ -1125,6 +1124,20 @@ Once this is done, restart `pfmon` or `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
+===== Disabling the automatic clustering resolution service (for PF versions 10.0 and later)
+
+You should disable `galera-autofix` service in configuration to prevent resolution of cluster issues during upgrade.
+
+In order to do, go in _Configuration->System Configuration->Services_ and disable the `galera-autofix` service.
```
Comment (julsemaan) [inline docs/PacketFence_Upgrade_Guide.asciidoc]:
```suggestion
In order to do so, go in _Configuration->System Configuration->Services_ and disable the `galera-autofix` service.
```

**Review by julsemaan — COMMENTED** [review]
A lot of content is duplicated but I only flagged the issues once, make sure you adjust both files based on my comments

**Comment by nqb** [comment]
Thanks for you review. All issues solved.

**Comment by julsemaan** [comment]
I'll let the doc kingpin (aka @extrafu) give his review on this PR and we can merge after

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 7]
```diff
@@ -1125,6 +1124,20 @@ Once this is done, restart `pfmon` or `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
+===== Disabling galera-autofix (for PF versions 10.0 and later)
```
Comment (extrafu) [inline docs/PacketFence_Upgrade_Guide.asciidoc]: PF -> PacketFence

Comment (extrafu) [reply docs/PacketFence_Upgrade_Guide.asciidoc]: versions -> version

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 8]
```diff
@@ -1544,15 +1540,18 @@ Once this is done, restart `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
-====== Restart the galera-autofix service
+===== Reactivate galera-autofix
 
-You now need to restart the galera-autofix service so that its aware that all the members of the cluster are online again.
+You now need to reactivate and restart the `galera-autofix` service so that its aware that all the members of the cluster are online again.
```
Comment (extrafu) [inline docs/PacketFence_Upgrade_Guide.asciidoc]: its -> it's

File: docs/cluster/appendix.asciidoc [hunk 9]
```diff
@@ -110,6 +110,20 @@ Once this is done, restart `pfmon` or `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
+==== Disabling galera-autofix (for PF versions 10.0 and later)
```
Comment (extrafu) [inline docs/cluster/appendix.asciidoc]: PF versions -> PacketFence version

File: docs/cluster/appendix.asciidoc [hunk 10]
```diff
@@ -405,16 +409,18 @@ Once this is done, restart `pfcron` on all nodes using:
 /usr/local/pf/bin/pfcmd service pfcron restart
 ----
 
-===== Restart the galera-autofix service
+==== Reactivate galera-autofix
 
+You now need to reactivate and restart the `galera-autofix` service so that its aware that all the members of the cluster are online again.
```
Comment (extrafu) [inline docs/cluster/appendix.asciidoc]: its -> it's

**Comment by extrafu** [comment]
All comments done, once modifications are applied, ready to be merge.

**Comment by nqb** [comment]
@extrafu, issues solved.

**Pull request merged**